### PR TITLE
updated Project MUSE.js new approach as suggested in https://github.com/zotero/translators/issues/2303

### PR DESCRIPTION
**Pull request by socheres**

### CODE REVIEW

File: Project MUSE.js [hunk 1]
```diff
@@ -188,75 +194,144 @@ var testCases = [
...
 				"url": "https://muse.jhu.edu/article/551992",
 				"volume": "49",
-				"attachments": [
+				"attachments": [],
+				"tags": [],
+				"notes": [
 					{
-						"title": "Full Text PDF",
-						"mimeType": "application/pdf"
+						"note": "<p>Volume 49, Number 2, 2014</p>"
```
Comment (adam3smith) [inline Project MUSE.js:259]: Discard the notes -- they don't contain additional information

File: Project MUSE.js [hunk 2]
```diff
@@ -86,100 +86,106 @@ function doWeb(doc, url) {
...
-	if (!abstract) abstract = ZU.xpathText(doc, '//div[@class="description"][1]');
-	if (!abstract) abstract = ZU.xpathText(doc, '//div[contains(@class, "card_summary") and contains(@class, "no_border")]');
-	var translator = Zotero.loadTranslator('web');
-	// Embedded Metadata
-	translator.setTranslator('951c027d-74ac-47d4-a107-9c3069ab7b48');
-	translator.setHandler('itemDone', function (obj, item) {
-		if (abstract) {
-			item.abstractNote = abstract.replace(/^\s*Abstract/, "").replace(/show (less|more)$/, "").replace(/,\s*$/, "")
-				.trim();
+	let citationURL = ZU.xpath(doc, '//*[contains(concat( " ", @class, " " ), concat( " ", "view_citation", " " ))]//a');
```
Comment (adam3smith) [inline Project MUSE.js:89]: why not just 
`ZU.xpathText(doc, '//li[@class="view-citation"]/a/@href')`
? And then you can just plug this right into the ZU.processDocuments function

**Review by adam3smith — COMMENTED** [review]
Thanks! A couple of small tweaks/requests

File: Project MUSE.js (hunk 1 above)
Comment (socheres) [reply Project MUSE.js]: thanks for the code review. yes, i discard it.

File: Project MUSE.js (hunk 2 above)
Comment (socheres) [reply Project MUSE.js]: yes. this is much simpler.

**Comment by adam3smith** [comment]
Thanks!

**Comment by adam3smith** [comment]
Remaining test discrepancy is authenticated vs. non-authenticated. The test represented the (better) authenticated data

**Pull request merged**